src/main/resources/i18n/Scalars.properties

```diff
@@ -0,0 +1,29 @@
+#
+# This resource bundle is used for the scalar code to produce i18n messages
+#
+# The keys have the format of rule class name and then message type within that.  Most rules
+# will only have 1 or 2 message keys
+#
+# Please try and keep this sorted within rule class and use # between sections so the IDEA Ctrl-Alt-L reformat does not bunch
+# them too tightly.
+#
+# REMEMBER - a single quote ' in MessageFormat means things that are never replaced within them
+# so use 2 '' characters to make it one ' on output.  This will take for the form ''{0}''
+#
+Scalar.unexpectedAstType=Expected an AST type of ''{0}'' but it was a ''{1}''.
+#
+Enum.badInput=Invalid input for enum ''{0}''. Unknown value ''{1}''.
+Enum.badName=Invalid input for enum ''{0}''. No value found for name ''{1}''
+Enum.unallowableValue=Literal value not in allowable values for enum ''{0}'' - ''{1}''.
+#
+Int.outsideRange=Expected value to be in the integer range, but it was a ''{0}''
+Int.notInt=Expected a value that can be converted to type 'Int' but it was a ''{0}''
+#
+ID.unexpectedAstType=Expected an AST type of ''IntValue'' or ''StringValue'' but it was a ''{0}''.
+ID.notId=Expected a value that can be converted to type 'ID' but it was a ''{0}''
+#
+Float.notFloat=Expected a value that can be converted to type 'Float' but it was a ''{0}''
+Float.unexpectedAstType=Expected an AST type of ''IntValue'' or ''FloatValue'' but it was a ''{0}''.
+#
+Boolean.notBoolean=Expected a value that can be converted to type 'Boolean' but it was a ''{0}''
```

Comment:
Nit: could remove the single quotes around `'Boolean'`, they won't be rendered

Reply:
fixed